src/connection_manager.c

```diff
@@ -157,6 +160,7 @@ static qd_config_sasl_plugin_t *qd_find_sasl_plugin(qd_connection_manager_t *cm,
     return 0;
 }
 
+// XXX Should this move to server.c or its own server_config.c?
 void qd_server_config_free(qd_server_config_t *cf)
 {
```

Comment:
Btw, I hate having to create new files. I always feel like I am polluting someone's else carefully thought out layout...

Bbtw, I thought that clang-format with the style selected would move all opening braces on the same line. Apparently not.